recipes/aws-c-cal/all/conanfile.py

```diff
@@ -97,5 +92,25 @@ def package_info(self):
             self.cpp_info.components["aws-c-cal-lib"].frameworks.append("Security")
         elif self.settings.os in ("FreeBSD", "Linux"):
             self.cpp_info.components["aws-c-cal-lib"].system_libs.append("dl")
+
+        self.user_info.with_openssl = self._needs_openssl
```

Comment:
Can you name it `with_openssl`? That name is more in-line with options of other recipes.